src/app/groups/containers/user-group-invitations/user-group-invitations.component.html

```diff
@@ -1,8 +1,85 @@
 <h2 class="alg-h2 alg-text-normal alg-base-title-space" i18n>Pending invitations</h2>
-<alg-pending-request
-  [state]="state"
-  [columns]="columns"
-  [requests]="requests"
-  (sort)="onFetch($event)"
-  (processRequests)="onProcessRequests($event)"
-></alg-pending-request>
+@if (state$ | async; as state) {
+  <alg-error
+    *ngIf="state.isError"
+    class="dark"
+    icon="ph-duotone ph-warning-circle"
+    i18n-message message="Error while loading the pending requests."
+  ></alg-error>
+  <p-table
+    class="alg-table"
+    [value]="state.data || []"
```

Comment:
in case of error, is the table is not displayed at all or does it look empty?

Reply:
Fixed